tests/TestClasses/Projectors/BalanceProjector.php

```diff
@@ -12,7 +12,7 @@ class BalanceProjector implements Projector
     use ProjectsEvents;
 
     protected $handlesEvents = [
-        MoneyAdded::class => 'onMoneyAdded',
+        MoneyAdded::class,
```

Comment:
Instead of editing `BalanceProjector` let make a dedicated test projector to make sure your changes word. Name if `ProjectorWithAssociativeAndNonAssociativeHandleEvents`.

I don't mind long class names. I'm open for suggestions if you can come up with a better name.

Reply:
Sounds good.  I had a test projector class with a long name, but opted to re-use `BalanceProjector`.  It had a long name, too.  ;-)

Reply:
@freekmurze Btw, I used "associative" and "indexed" for some method names (instead of "associative" and "non-associative").  I don't care one way or the other, but I'd prefer to be consistent.  Do you have a preference...?  "Non-associative" or "indexed"...?

Reply:
FYI...I've decided to go with "non-associative", so I've updated the private method name that I added earlier to keep in line with the same terms that you've used.